apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -106,6 +106,9 @@ export function TokenRow({
 
 	const isRenderSwapButton = allowedSwapCoinsList.includes(coinType);
 
+	// TODO: add balanceInUsd API call
+	const balanceInUsd = 0;
```

Comment:
pending on whether or not we are getting them as `int` or `bigint` here. Can modify the code